bioio_nd2/reader.py

```diff
@@ -109,6 +111,19 @@ def physical_pixel_sizes(self) -> types.PhysicalPixelSizes:
             with nd2.ND2File(f) as rdr:
                 return types.PhysicalPixelSizes(*rdr.voxel_size()[::-1])
 
+    @property
+    def binning(self) -> str | None:
+        """
+        Returns
+        -------
+        binning : str | None
+            Binning value reported by the ND2File metadata, e.g., "1x1".
+        """
+        with self._fs.open(self._path, "rb") as f, nd2.ND2File(f) as rdr:
```

Comment:
How close do these `binning` values align with the `binning` values we track in FMS currently?

Reply:
I have only ever seen '1x1'